components/blog/card/index.js

```diff
@@ -141,7 +141,7 @@ const StyledHeading = styled(Heading)`
   z-index: 2;
 
   &:hover {
-    color: #00ccbc;
+    color: ${({ type }) => (type === 'hero' ? '#f2f1f1' : '#00ccbc')};
```

Comment:
@majid-amiri During this refactor we should move to theme Variables perhaps, most colours should adapt from the context colours.

Reply:
Agree - Was mad with stylelint errors and didn't notice this one :)